src/__screenshot_tests__/navigation-bar-screenshot-test.tsx

```diff
@@ -16,7 +16,7 @@ test.each`
 
     const image = await page.screenshot();
 
-    expect(image).toMatchImageSnapshot();
+    expect(image).toMatchImageSnapshot({failureThreshold: 0.00001});
```

Comment:
This test seems unstable because of the small picture:

![unstable](https://github.com/Telefonica/mistica-web/assets/25785151/612b5789-c53f-4906-a2d9-f0cade48aab6)